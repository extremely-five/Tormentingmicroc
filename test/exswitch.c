void main(int n){
     int i;
     i=1;
     switch(n){
        case 0:print (1);
        case 1:print (2);
        default:print (3);
     }
    
}