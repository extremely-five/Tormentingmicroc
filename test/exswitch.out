24 19 1 5 25 15 1 13 0 1 1 0 1 12 15 -1 13 11 18 27 0 1 22 15 -1 16 34 13 11 0 1 6 17 41 0 2 22 15 -1 16 48 13 11 13 11 6 17 53 0 3 22 21 2 21 1